client/src/components/modal-forms/ManageGraduationForm.js

```diff
@@ -34,18 +34,49 @@ export const ManageGraduationForm = ({ initialValues, onClose, onSubmit }) => {
                     },
                     {
                       value: 'failed_cohort',
-                      label: 'Failed',
+                      label: 'Unsuccessful cohort',
                     },
                   ]}
                 />
                 {
-                  <DependantDateForm
-                    display={values.status === 'post_secondary_education_completed'}
+                  <Field
                     test-id={'editGraduationModalStatus'}
-                    name={'data.graduationDate'}
-                    label={'Graduation Date'}
+                    name={'data.date'}
+                    label={
+                      values.status === 'post_secondary_education_completed'
+                        ? 'Graduation Date'
+                        : 'Unsuccesful Graduation Date'
+                    }
+                    component={RenderDateField}
+                    maxDate={getTodayDate()}
                   />
                 }
+                {values.status === 'failed_cohort' && (
+                  <Field
+                    test-id={'editGraduationModalRehire'}
+                    name='rehire'
+                    component={RenderRadioGroup}
+                    label='I intend on rehiring for this position'
+                    options={[
+                      {
+                        value: 'rehire_yes',
+                        label: 'Yes',
+                      },
+                      {
+                        value: 'rehire_no',
+                        label: 'No',
+                      },
+                    ]}
+                  />
+                )}
+                {values?.rehire === 'rehire_no' && (
+                  <Field
+                    test-id={'editGraduationModalWithdraw'}
+                    name='withdraw'
+                    component={RenderCheckbox}
+                    label='Withdraw this participant from the program.'
```

Comment:
Doesn't match design copy

"Please use the archive function to withdraw this participant from the HCAP program."

Reply:
Wait did you go above and beyond? It looks like you're actually completing the withdrawal here. That wasn't required for the user story 😅 if it's done we need to update the AC as the test plan won't expect this

Reply:
@fwkendall, the logic is already implemented.

Reply:
We need to inform Akhil and Tavy then, since the AC and test plan don't cover this